### DISCUSSION TITLE
Is paddleocr impacted by CVE-2007-4559, a python vulnerability in the tarfile module?

### DISCUSSION THREAD

**Discussion by rohit049** [PaddlePaddle/PaddleOCR#7770] [Q&A] [answered]
The Python `tarfile` module has a very old vulnerability that surfaced over the news release.
Is this vulnerability present for paddleocr?

**Comment by WenmuZhou** [comment, marked as answer]
We did not use tarfile.extract() or tarfile.extractall() functions
